database/migration/migrateData.ts

```diff
@@ -583,6 +584,25 @@ const migrateData = async () => {
     ALTER TABLE data_table
     ADD COLUMN data_view_code varchar;
     `)
+
+    console.log(' - Update existing review_assignment to set is_locked=false when isSelfAssigned')
+    // TODO - Not sure how to do this
+
+    // This logs is related to something which will be added by view_functions_triggers file
+    console.log(' - Add new generated column available_sections in review_assignment TABLE')
+
+    console.log(' - Remove type ASSIGN_LOCKED from assign_action ENUM')
+    await DB.changeSchema(`
+      ALTER TYPE public.assigner_action RENAME to public.assigner_action_old;
+
+      CREATE TYPE public.assigner_action AS ENUM
+      (
+        'ASSIGN',
+        'RE_ASSIGN'
+      );
+
+      DROP TYPE public.assigner_action_old;
```

Comment:
This won't work as it's still in use.

Unfortunately you have to add commands to change the columns of the old one to the new one. See how I did it for the trigger values -- around line 355 of this file.